src/renderer/components/+workloads-pods/__tests__/pod-container-env.test.tsx

```diff
@@ -0,0 +1,273 @@
+      });
+    });
+
+    const container: PodContainer = {
+      image: "my-image",
+      name: "my-first-container",
+      envFrom: [{
+        configMapRef: {
+          name: "my-config-map",
+        },
+      }],
+    };
+    const pod = new Pod({
+      apiVersion: "v1",
+      kind: "Pod",
+      metadata: {
+        name: "my-pod",
+        namespace: "default",
+        resourceVersion: "1",
+        selfLink: "/api/v1/pods/default/my-pod",
+        uid: "1234",
+      },
+      spec: {
+        containers: [container],
+      },
+    });
+    const result = render(<ContainerEnvironment container={container} namespace={pod.getNs()} />);
+
+    expect(result.getByTestId("envFrom-configmap-my-config-map").innerHTML).toBe(`<span class="var-name">configFoo</span>= configBar`);
+  });
+
+  it("renders envFrom when given a secretRef", () => {
+    secretStore.getByName.mockImplementation((name, namespace) => {
+      expect(name).toBe("my-secret");
+      expect(namespace).toBe("default");
+
+      return new Secret({
+        apiVersion: "v1",
+        kind: "Secret",
+        metadata: {
+          name: "my-secret",
+          namespace: "default",
+          resourceVersion: "3",
+          selfLink: "/api/v1/secrets/default/my-secret",
+          uid: "237",
+        },
+        type: SecretType.BasicAuth,
+        data: {
+          bar: "bat",
+        },
+      });
+    });
+
+    const container: PodContainer = {
+      image: "my-image",
+      name: "my-first-container",
+      envFrom: [{
+        secretRef: {
+          name: "my-secret",
+        },
+      }],
+    };
+    const pod = new Pod({
+      apiVersion: "v1",
+      kind: "Pod",
+      metadata: {
+        name: "my-pod",
+        namespace: "default",
+        resourceVersion: "1",
+        selfLink: "/api/v1/pods/default/my-pod",
+        uid: "1234",
+      },
+      spec: {
+        containers: [container],
+      },
+    });
+    const result = render(<ContainerEnvironment container={container} namespace={pod.getNs()} />);
+
+    expect(result.getByTestId("envFrom-secret-my-secret-bar").innerHTML).toMatch(`<span class="var-name">bar</span>= secretKeyRef(my-secret.bar)`);
+  });
+
+  it("renders env", () => {
+    const container: PodContainer = {
+      image: "my-image",
+      name: "my-first-container",
+      env: [{
+        name: "foobar",
+        value: "https://localhost:12345",
+      }],
+    };
+    const pod = new Pod({
+      apiVersion: "v1",
+      kind: "Pod",
+      metadata: {
+        name: "my-pod",
+        namespace: "default",
+        resourceVersion: "1",
+        selfLink: "/api/v1/pods/default/my-pod",
+        uid: "1234",
+      },
+      spec: {
+        containers: [container],
+      },
+    });
+    const result = render(<ContainerEnvironment container={container} namespace={pod.getNs()} />);
+
+    expect(result.getByTestId("env-foobar").innerHTML).toBe(`<span class="var-name">foobar</span>= https://localhost:12345`);
```

Comment:
Hmm. Could it be bit less technical? 

```
expect(result.getByTestId("env-foobar")).toHaveTextContent("foobar= https://localhost:12345");
```

Reply:
Yes, I completely forgot about those extended `expect` functions.